code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -53,16 +53,23 @@
 	assignedrole = "Ash Walker"
 	var/datum/team/ashwalkers/team
 
+/obj/effect/mob_spawn/human/ash_walker/allow_spawn(mob/user)
+	if(user.key in team.players_spawned)//if player is on exaust list
+		to_chat(user, "<span class='warning'><b>You have exausted your usefulness to the Necropolis</b>.</span>")
+		return FALSE
+	return TRUE
+
 /obj/effect/mob_spawn/human/ash_walker/special(mob/living/new_spawn)
 	new_spawn.fully_replace_character_name(null,random_unique_lizard_name(gender))
-	to_chat(new_spawn, "<b>Drag the corpses of men and beasts to your nest. It will absorb them to create more of your kind. Don't leave your nest undefended, protect it with your life. Glory to the Necropolis!</b>")
+	to_chat(new_spawn, "<b>Drag the corpses of men and beasts to your nest. It will absorb them to create more of your kind. Invade the strange structure of the outsiders if you must. Do not cause unnecessary destruction, as littering the wastes with ugly wreckage is certain to not gain you favor. Glory to the Necropolis!</b>")
```

Comment:
might wanna specify that just kidnapping people from the station and sacrificing them to the nest doesn't count as "unnecessary destruction". sure, it's _implied_, but it's best to be as clear as possible about these sorts of things

Reply:
Seems to me that's handled in the first sentence, "Drag the corpses of men and beast to your nest". I suppose it does say corpses, but every corpse was at one time living.